docs/developing.md

```diff
@@ -37,19 +40,19 @@ Tests are executed with `pytest`. You can run them in parallel with for instance
 pytest -n 5
 ```
 
-Build the `jupytext` package and install it with
+Some tests require a Jupyter kernel pointing to the current environment:
 ```
-pip install -U build wheel
-python -m build
-pip install dist/jupytext-x.x.x-py3-none-any.whl
+python -m ipykernel install --name jupytext-dev --user
 ```
 
-or with
+Finally, you can build the package and install it with
 ```
-pip install .
+pip install -U build wheel
+python -m build
+pip install dist/jupytext-x.x.x-py3-none-any.whl
```

Comment:
Just out of curiosity, why build a `wheel` and install it instead of directly doing a `pip install .`?

Reply:
Well I am afraid it just tells that I know so little about wheels and tars etc. Before `jupytext==1.16` I would use that to test the extension because the editable install would not include the extension, but from your comment it seems that I can remove that part, so I'll remove it!

Reply:
Ahh, ok I see. Yes, now we build the extension _always_, so we can directly do editable installs!